doc/user/zebra.rst

```diff
@@ -1241,36 +1241,103 @@ For protocols requiring an IPv6 router-id, the following commands are available:
 
 .. _zebra-sysctl:
 
-Expected sysctl settings
+sysctl settings
 ========================
 
 The linux kernel has a variety of sysctl's that affect it's operation as a router.  This
 section is meant to act as a starting point for those sysctl's that must be used in
 order to provide FRR with smooth operation as a router.  This section is not meant
 as the full documentation for sysctl's.  The operator must use the sysctl documentation
-with the linux kernel for that.
+with the linux kernel for that. The following link has helpful refernces to many relevant
+sysctl values:  https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt
+
+Expected sysctl settings
+^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. option:: net.ipv4.ip_forward = 1
 
-   This option allows the linux kernel to forward ipv4 packets incoming from one interface
-   to an outgoing interface.  Without this no forwarding will take place from off box packets.
+   This global option allows the linux kernel to forward (route) ipv4 packets incoming from one
+   interface to an outgoing interface.  If this is set to 0, the system will not route transit
```

Comment:
double space before 'If'

Reply:
There are both single and double spaces following a period all throughout zebra.rst, both inside and outside the sysctl section.
Is there a formatting standard that we adhere to?  would like to make this consistent if possible